src/main/kotlin/racingcar/ui/RaceGameController.kt

```diff
@@ -3,21 +3,30 @@ package racingcar.ui
 import racingcar.domain.Car
 import racingcar.domain.Cars
 import racingcar.domain.Race
-import racingcar.domain.RaceResult
-import racingcar.domain.strategy.RandomMovableStrategy
+import racingcar.ui.view.getCarNames
+import racingcar.ui.view.getTotalRound
+import racingcar.ui.view.printRaceResult
 
 object RaceGameController {
-    fun readyForRace(carNames: List<String>, totalRound: Int): Race {
-        val cars = Cars(carNames.map { Car(it) })
+    fun play() {
+        val race = readyForRace()
+        val raceResult = race.run(movableStrategy = {
+            RANDOM_NUMBER_RANGE.random() >= MOVABLE_CONDITION_NUMBER
+        })
```

Comment:
메서드의 가장 마지막 파라미터가 함수식이라면 람다식으로 대체할 수 있어요. 😃

```suggestion
        val raceResult = race.run { RANDOM_NUMBER_RANGE.random() >= MOVABLE_CONDITION_NUMBER }
```

Reply:
MovableStrategy을 구현한 `RandomMovableStrategy` 구현체를 만들어서 `MOVABLE_CONDITION_NUMBER`, `RANDOM_NUMBER_RANGE`와 관련된 세부 사항들을 구현체 내부에 두는 것이 더 좋지 않을까요? 🤔 

아무래도 [저번 코멘트](https://github.com/next-step/kotlin-racingcar/pull/935#discussion_r1057294896)의 내용 때문에 비즈니스 코드 영역에서도 람다식으로 수정하신 듯 합니다. 🥲 
구현부에서는 이전 방식이 더 캡슐화가 잘돼있다고 생각해서 이전 방식처럼 쓰시는 것이 더 좋은 것 같습니다! 

테스트 코드를 위한 Stub을 만들필요가 없다는 점에서 저번 코멘트를 드린거였는데 오해의 소지가 있었던 것 같아요. 🙏